core/src/test/java/com/oath/oak/OakMemoryManagerTest.java

```diff
@@ -53,325 +53,4 @@ public void allocate() {
         assertEquals(4, bb.remaining());
         assertEquals(8, memoryManager.allocated());
     }
```

Comment:
Why did you remove unit tests?

Reply:
because they dont mean anything anymore. they check when the release list is freed but in this new implementation there is no release list